README.md

```diff
@@ -110,38 +105,37 @@ When contributing to **Captable, Inc.**, whether on GitHub or in other community
 - Copy `.env.example` to `.env`
 
   ```bash
-  cp .env.example .env`
+  cp .env.example .env
   ```
 
 - Run the following command to start the development environment
 
   ```bash
-  
+
   # With pnpm installed
   pnpm dx
 
   # Without pnpm installed
   docker compose up
-  
+
   ```
 
 - Run the following command to migrate and seed the database
 
   ```bash
-  
+
   docker compose run app pnpm db:migrate
   docker compose run app pnpm db:seed
 
   ```
 
-  > **Note**
```

Comment:
Are you deleting this line on purpose?

Reply:
Yea, I deleted that line on purpose, because if we get rid of `Dockerfile.dev`, `docker-compose build` is no longer relevant.